apps/admin/webpack.mocked.config.js

```diff
@@ -0,0 +1,31 @@
+
+const path = require('path');
+const { ProvidePlugin } = require('webpack');
+// In non-live dev mode, we redirect imports to mocked versions
+const projectRoot = path.resolve(__dirname, '..', '..');
+const mocksFolder = path.resolve(projectRoot, '__mocks__');
+
+module.exports = {
+  module: {
+    rules: [
+      // Allow ts-loader to parse mocks
+      {
+        test: /\.ts(x?)$/,
+        include: mocksFolder,
+        use: {
+          loader: 'ts-loader',
+          options: {
+            configFile: path.join(projectRoot, 'tsconfig.base.json'),
+            transpileOnly: true,
+            experimentalWatchApi: true,
+          },
+        },
+      }
+    ],
+  },
+  // Re-use our jest mocks inside our website (neat, huh?)
+  resolve: {
+    conditionNames: [process.env.CONFIG_NAME, "electron", "browser", "require", "default"],
+    modules: [mocksFolder]
+  }
```

Comment:
This is obvs not ready for production yet

Reply:
I'm working on that now.  Just trying to figure out exactly how handle the switching.

We now have webpack/node/jest & this add's Electron into our build config shenanigans.  It's not super-fun to balance them all.